### ISSUE TITLE
Not working?

### ISSUE THREAD

**Issue by cacack** [terrencepreilly/darglint#62]
I don't see anyone talking about this so it has to be something I'm doing wrong...

Using pyenv to provide 3.8.1 with pipenv.

With a file containing:
```
def tryme():
    pass
```

Running darglint doesn't complain...

```
$ pipenv install --dev darglint
$ pipenv run darglint dummy.py
$ echo $?
0
```

In my real project I also have flake8 installed and it notes that darglint is installed, but also doesn't complain about the lack of docstrings.

**Comment by rpkilby** [comment]
iirc, docstrings are optional, and darglint will only validate them if present.

**Comment by terrencepreilly** [comment]
@rpkilby is correct.  Darglint tries not interfere with [pydocstyle](https://github.com/pycqa/pydocstyle) (which throws errors for missing docstrings.)
